tests/osgi-tests/src/test/resources/wso2-carbon-datasources.xml

```diff
@@ -5,10 +5,10 @@
             <description>The datasource used for registry and user manager</description>
             <definition type="RDBMS">
                 <configuration>
-                    <url>jdbc:mysql://${docker.container.ip}/WSO2_ANALYTICS_DB</url>
+                    <url>jdbc:h2:./target/dasdb</url>
                     <username>root</username>
                     <password>root</password>
-                    <driverClassName>com.mysql.jdbc.Driver</driverClassName>
+                    <driverClassName>jdbc:h2:./target/dasdb</driverClassName>
```

Comment:
Is this correct? Looks like we should have "org.h2.Driver" here instead.

Reply:
It was wrong. Fixed in e6d2373